docs/nidmm/functions.rst

```diff
@@ -63,15 +63,15 @@ NI-DMM Functions
         :py:data:`nidmm.function` attribute to AC measurements. The driver sets the
         :py:data:`nidmm.AC\_MAX\_FREQ` attribute to this value. The valid range is 1
         Hz–300 kHz for the NI 4080/4081/4082 and the NI 4070/4071/4072, 10
-        Hz–100 Hz for the NI 4065, and 20 Hz–25 kHz for the NI 4050 and NI 4060.
+        Hz–100 Hz for the NI 4065, and 20 Hz–25 kHz for the NI 4050 and NI 4060.
```

Comment:
To make it work with Python2, I change non-breaking spaces '\xa0' to normal spaces.